### ISSUE TITLE
bug in attaching evidence for extensions in BP only form

### ISSUE THREAD

**Issue by krchristie** [geneontology/noctua-form#225]
The BP form does not seem to attach evidence properly to at least some extensions. Here is the example I found:

**I selected the "occurs in **Anatomy**" option to add a cell type extension:**

![20230621-CCform-evForExtension-p1](https://github.com/geneontology/noctua-form/assets/10533218/37c5d07c-cb88-43e2-b08b-cbe67a89d520)

**I filled in the new row for the extension and clicked "Save":**
![20230621-CCform-evForExtension-p2](https://github.com/geneontology/noctua-form/assets/10533218/e8b2d2df-9eff-4964-b96a-5ca1f9d016af)

**Since the form doesn't show everything, you don't see that the extension is missing the evidence.:**
![20230621-CCform-evForExtension-p3](https://github.com/geneontology/noctua-form/assets/10533218/c36d6f24-9158-4154-b9ad-1da6fc4e71ac)

**However, in you go to the graph, you see that the evidence is missing for the extension.:**
![20230621-CCform-evForExtension-p4](https://github.com/geneontology/noctua-form/assets/10533218/8991585e-9d51-4f44-8925-a6e4e713fcb2)

**The Annotation Preview confirms that the annotation extension is missing.:**
![20230621-CCform-evForExtension-p5](https://github.com/geneontology/noctua-form/assets/10533218/f78c5fda-97f2-4772-ab3d-a5f2a96ad751)

**Comment by vanaukenk** [comment]
At the Noctua workbenches call (2023-06-22), we reproduced the behavior (i.e. missing evidence on occurs_in BP extension in the BP only form).

We also tested the equivalent extension in the Activity Unit form and, there, the evidence IS properly attached.

We also tested an extension in the CC only form and the evidence was also attached properly there.

@tmushayahama would like to systematically check each extension to see if evidence is correctly saved.
